sdk/communication/Azure.Communication.Sms/src/SmsClient.cs

```diff
@@ -117,37 +121,49 @@ public virtual async Task<Response<SendSmsResponse>> SendAsync(PhoneNumberIdenti
+            Argument.AssertNotNullOrEmpty(from, nameof(from));
+            Argument.AssertNotNullOrEmpty(to, nameof(to));
+            Response<IEnumerable<SmsSendResult>> allSmsSendResults = Send(from, new[] { to }, message, options, cancellationToken);
+            SmsSendResult sendSMSResult = allSmsSendResults.Value.FirstOrDefault();
+            return Response.FromValue(sendSMSResult, allSmsSendResults.GetRawResponse());
         }
 
         /// <summary> Sends an SMS message from a phone number that belongs to the authenticated account. </summary>
         /// <param name="from"> The sender&apos;s phone number in E.164 format that is owned by the authenticated account. </param>
-        /// <param name="to"> The recipient&apos;s phone number in E.164 format. In this version, only one recipient in the list is supported. </param>
+        /// <param name="to"> The recipient&apos;s phone number in E.164 format. In this version, up to 100 recipients in the list is supported. </param>
         /// <param name="message"> The contents of the message that will be sent to the recipient. The allowable content is defined by RFC 5724. </param>
-        /// <param name="sendSmsOptions"> Optional configuration for sending SMS messages. </param>
+        /// <param name="options"> Optional configuration for sending SMS messages. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="from"/> is null.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="to"/> is null.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="message"/> is null.</exception>
-        public virtual async Task<Response<SendSmsResponse>> SendAsync(PhoneNumberIdentifier from, IEnumerable<PhoneNumberIdentifier> to, string message, SendSmsOptions sendSmsOptions = null, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<IEnumerable<SmsSendResult>>> SendAsync(string from, IEnumerable<string> to, string message, SmsSendOptions options = default, CancellationToken cancellationToken = default)
         {
             using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(SmsClient)}.{nameof(Send)}");
             scope.Start();
             try
             {
-                Argument.AssertNotNullOrEmpty(from.PhoneNumber, nameof(from));
-                return await RestClient.SendAsync(from.PhoneNumber, to.Select(x => AssertNotNullOrEmpty(x.PhoneNumber, nameof(to))), message, sendSmsOptions, cancellationToken).ConfigureAwait(false);
+                Argument.AssertNotNullOrEmpty(from, nameof(from));
+                List<SmsRecipient> smsRecipients = new List<SmsRecipient>();
```

Comment:
IEnumerable<SmsRecipient> recipients = to.Select(x =>
    new SmsRecipient(x)
    {
        RepeatabilityRequestId = Guid.NewGuid().ToString(),
        RepeatabilityFirstSent = DateTime.Now.ToString(CultureInfo.InvariantCulture),
    });